modules/diskq/qdisk.c

```diff
@@ -618,10 +634,16 @@ _load_state(QDisk *self, GQueue *qout, GQueue *qbacklog, GQueue *qoverflow)
 
       msg_info("Disk-buffer state loaded",
                evt_tag_str("filename", self->filename),
-               evt_tag_long("qout_length", self->hdr->qout_pos.count),
-               evt_tag_long("qbacklog_length", self->hdr->qbacklog_pos.count),
-               evt_tag_long("qoverflow_length", self->hdr->qoverflow_pos.count),
-               evt_tag_long("qdisk_length", self->hdr->length));
+               evt_tag_long("number_of_messges", _number_of_messages(self)));
+
+      msg_debug("Disk-buffer state loaded",
```

Comment:
could you use a different "message" than the one on msg_info() printed just before this one?

Reply:
Suggestion:

info: "Diisk-buffer state loaded"
debug: "Disk-buffer internal state"